.env.development.local

```diff
@@ -0,0 +1 @@
+FAST_REFRESH=true
```

Comment:
@berteotti does this solve #1019 for you?

Reply:
It's perfect now! This solves it.